src/headingsengine.js

```diff
@@ -18,11 +18,25 @@ const formats = [
 	{ id: 'heading3', viewElement: 'h4', label: 'Heading 3' }
 ];
 
+/**
+ * The headings feature. Handles switching between block formats - different headings and paragraph.
+ * This class represents part of the feature that can be used without any UI - for example inside
```

Comment:
I wouldn't write this so explicitly here that engine part can be used in Node.js. I'd keep it general that: "This feature represents the engine part of the Headings feature."